bin/create-layer-templates

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { createLayerTemplate, createVariantTemplate } = require('../dist/main');
+
+const argv = yargs(hideBin(process.argv))
+  .usage('Usage: $0 indir outdir')
+  .option('in-dir', {
+    alias: 'inDir',
+    default: 'styles',
+    describe: 'The input style directory',
+    type: 'string'
+  })
+  .option('out-dir', {
+    alias: 'outDir',
+    default: 'templates',
+    describe: 'The output directory',
+    type: 'string'
+  })
+  .option('base-path', {
```

Comment:
Can we call this `base-style-path` instead? `base-path` sounds like all operations are going to start from that path, which isn't the case.